### PR TITLE
feat: add cliphist integraton

### LINKED ISSUE #83: add a clipboard manager
just saw it should be pretty straightforward to add one using[ wl-clipboard](https://github.com/bugaevc/wl-clipboard):

```bash
exec wl-paste -t text --watch clipman store --max-items=60 --histpath="~/.local/share/clipman.json"
bindsym $mod+h exec clipman pick --tool="wofi" --max-items=30 --histpath="~/.local/share/clipman.json"
```

### PR DESCRIPTION
**Pull request by boredland**
closes https://github.com/Manjaro-Sway/manjaro-sway/issues/83

### CODE REVIEW

File: community/sway/etc/sway/modes/default [hunk 1]
```diff
@@ -11,6 +11,9 @@ $bindsym $mod+Shift+q kill
 ## Launch // Launcher ##
 $bindsym $mod+d exec $menu
 
+## Launch // Clipboard ##
+$bindsym $mod+Shift+P $clipboard
```
Comment (boredland) [inline community/sway/etc/sway/modes/default]:
```suggestion
$bindsym $mod+Shift+p exec $clipboard
```

**Review by Mathew-D — APPROVED** [review]
Looks good.  I like how you moved the  $rofi_theme setup. I will give it a try on the git version once it is merged and I get home.

**Review by Mathew-D — CHANGES_REQUESTED** [review]
Why:
 "exec": "cliphist list | wc -l",
        "exec-if": "[ $(cliphist list | wc -l) -gt 0 ]"
Do we really want the number of items showing up.  Why not just put that on tooltip instead?

**Comment by boredland** [comment]
> Why:
> "exec": "cliphist list | wc -l",
> "exec-if": "[ $(cliphist list | wc -l) -gt 0 ]"
> Do we really want the number of items showing up. Why not just put that on tooltip instead?

you're right, tooltip is enough.

**Comment by Mathew-D** [comment]
There doesn't seem to be a way to set the tooltip to the output for the exec.

File: community/sway/usr/share/sway/templates/waybar/config.jsonc [hunk 2]
```diff
@@ -254,5 +255,12 @@
         "on-scroll-up": "playerctl position 10+; pkill -RTMIN+5 waybar",
         "on-scroll-down": "playerctl position 10-; pkill -RTMIN+5 waybar",
         "signal": 5
+    },
+    "custom/clipboard": {
+        "format": " {}",
+        "tooltip": false,
+        "on-click": "swaymsg -q exec '$clipboard'",
+        "exec": "cliphist list | wc -l",
```
Comment (boredland) [inline community/sway/usr/share/sway/templates/waybar/config.jsonc]:
```suggestion
        "exec": "printf '{\"tooltip\":\"%s\"}' $(cliphist list | wc -l)",
```

File: community/sway/usr/share/sway/templates/waybar/config.jsonc [hunk 3]
```diff
@@ -254,5 +255,12 @@
         "on-scroll-up": "playerctl position 10+; pkill -RTMIN+5 waybar",
         "on-scroll-down": "playerctl position 10-; pkill -RTMIN+5 waybar",
         "signal": 5
+    },
+    "custom/clipboard": {
+        "format": " {}",
+        "tooltip": false,
```
Comment (boredland) [inline community/sway/usr/share/sway/templates/waybar/config.jsonc]:
```suggestion
      "return-type": "json",
```

File: community/sway/usr/share/sway/templates/waybar/config.jsonc [hunk 4]
```diff
@@ -254,5 +255,12 @@
         "on-scroll-up": "playerctl position 10+; pkill -RTMIN+5 waybar",
         "on-scroll-down": "playerctl position 10-; pkill -RTMIN+5 waybar",
         "signal": 5
+    },
+    "custom/clipboard": {
+        "format": " {}",
```
Comment (boredland) [inline community/sway/usr/share/sway/templates/waybar/config.jsonc]:
```suggestion
        "format": "",
```

**Comment by boredland** [comment]
> There doesn't seem to be a way to set the tooltip to the output for the exec.

seems to work using the json return.

**Review by Mathew-D — APPROVED** [review]
Great job.

**Pull request merged**